app/pods/publications/publication/template.hbs

```diff
@@ -18,106 +21,107 @@
-                          @icon ="calendar"
-                          value={{this.getPublicationBeforeDate}}
-                    {{on "blur" this.setPublicationBeforeDate}}>
-                    <WebComponents::AuIcon @name="calendar"/>
-                  </WebComponents::AuDatepicker>
-                </div>
-                <div class="auk-form-group">
-                  <WebComponents::AuLabel>{{t 'extreme-translation-date'}}</WebComponents::AuLabel>
-                  <WebComponents::AuDatepicker
-                          @block="true"
-                          @icon ="calendar"
-                          value={{this.getTranslationDate}}
-                    {{on "blur" this.setTranslationDate}}>
-                    <WebComponents::AuIcon @name="calendar"/>
-                  </WebComponents::AuDatepicker>
-                </div>
+  <div class="auk-panel-layout__sidebar">
+    <WebComponents::AuSidebar @position="right" class={{if
+      this.collapsed
+      "auk-sidebar--collapsed auk-panel-layout--large auk-sidebar--responsive"
+      "auk-panel-layout__sidebar auk-sidebar--large auk-sidebar--responsive"
+    }}>
+      <WebComponents::AuNavbar @border="bottom" @skin="gray-100">
+        <WebComponents::AuToolbar>
+          <WebComponents::AuToolbar::Group @position="left"/>
+          <WebComponents::AuToolbar::Group @position="right">
+            <WebComponents::AuToolbar::Item>
+              <WebComponents::AuButton @skin="borderless" @icon={{if this.collapsed "chevron-left" "chevron-right"}}
+                                       @layout="icon-only" {{on 'click' this.toggleSidebar}} />
+            </WebComponents::AuToolbar::Item>
+          </WebComponents::AuToolbar::Group>
+        </WebComponents::AuToolbar>
+      </WebComponents::AuNavbar>
+      {{#if (not collapsed)}}
+        <div class="u-m-3">
+          <div class="auk-form-group">
+            <WebComponents::AuLabel>{{t 'publications-number'}}</WebComponents::AuLabel>
+            <WebComponents::AuInput type="number" value={{this.getPublicationNumberSidePanel}} @block="true"
+              {{on "change" this.setPublicationNumber}} />
+          </div>
+          <div class="auk-form-group">
+            <WebComponents::AuLabel for="blockInput">{{t 'status'}}</WebComponents::AuLabel>
+            <WebComponents::AuSelect @block="true" {{on "change" this.setPublicationStatus}}>
+              <option>{{t "to-publish"}}</option>
+              <option>{{t "published"}}</option>
+            </WebComponents::AuSelect>
+          </div>
+          <div class="auk-form-group">
+            <WebComponents::AuLabel for="blockInput">{{t "kind-of-publication"}}</WebComponents::AuLabel>
+            <WebComponents::AuSelect @block="true" {{on "blur" this.setPublicationForm}}>
+              <option>{{t "publication-from-extenso"}}</option>
+              <option>{{t "publication-form-by-except"}}</option>
+            </WebComponents::AuSelect>
```

Comment:
Hier moet je de ID kunnen doorgeven aan de optie zodat je dat in de `onChange` kunt afhandelen.